install.js

```diff
@@ -8,6 +8,7 @@ connection.query(
 	'points INT,' +
 	'num_lines INT,' +
 	'level INT,' +
+	'pf VARCHAR(200),' +
```

Comment:
I just left it there without thinking about it, but yea it should be changed, so I'll update it in the next commit. 
I also think that we can guarantee that it is < 200, because the max length of a Twitch username is 25 chars and the random generated string isn't that long either. I think the max length is 115, but I set it to 200 just to be sure.